testsuite/integration-arquillian/tests/base/src/main/java/org/keycloak/testsuite/arquillian/containers/AbstractQuarkusDeployableContainer.java

```diff
@@ -238,6 +238,9 @@ protected void addFeaturesOption(List<String> commands) {
             }
         }
 
+        commands.removeIf("--optimized"::equals);
+        commands.add("--http-relative-path=/auth");
```

Comment:
Do we need these two options in the testsuite?

Reply:
Once we change features, we need to rebuild the image. Once you rebuild the image, you need to-add the build time parameters, which was the `http-relative-path`. There is a similar snippet of code in the same file.  

Apparently this is the first time we're changing features on the command line?

So TL;DR: Yes.